### PR TITLE
miro.xml

### PR DESCRIPTION
**Pull request by Tobias-B-Besemer**

### CODE REVIEW

File: cleaners/miro.xml [hunk 1]
```diff
@@ -39,5 +51,10 @@
     <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro.log"/>
     <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro.log.1"/>
     <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro_Download_Daemon_a.txt"/>
+    <!-- Polish Windows XP Support: -->
+    <action command="delete" search="file" path="$USERPROFILE\Ustawienia lokalne\Temp\Miro-downloader.log"/>
```
[the remark this answers is not in the document]
Comment (Tobias-B-Besemer) [reply cleaners/miro.xml]: OK, will do!

[the remark this answers is not in the document]
Comment (Tobias-B-Besemer) [reply cleaners/miro.xml]: I guess I had a better idea... I just use `$Temp`... ;-)

[the remark this answers is not in the document]
Comment (Tobias-B-Besemer) [reply cleaners/miro.xml]: ...also adds support for Windows Vista and higher... ;-)

File: cleaners/miro.xml [hunk 2]
```diff
@@ -18,26 +18,48 @@
...
-    <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro.log"/>
-    <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro.log.1"/>
-    <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro_Download_Daemon_a.txt"/>
+    <action command="delete" search="file" path="$Temp\Miro-downloader.log"/>
+    <action command="delete" search="file" path="$Temp\Miro.log"/>
+    <action command="delete" search="file" path="$Temp\Miro.log.1"/>
+    <action command="delete" search="file" path="$Temp\Miro_Download_Daemon_a.txt"/>
+  </option>
+  <option id="temp">
+    <label>Temporary files</label>
```
Comment (az0) [inline cleaners/miro.xml]: Lines 58-59 are translated. Good

File: cleaners/miro.xml [hunk 3]
```diff
@@ -18,26 +18,48 @@
...
+    <description>Delete the temporary files</description>
+    <action command="delete" search="file" path="$Temp\Miro-downloader.log"/>
+    <action command="delete" search="file" path="$Temp\Miro.log"/>
+    <action command="delete" search="file" path="$Temp\Miro.log.1"/>
```
Comment (az0) [inline cleaners/miro.xml]: Lines 60-62 deletes three log files that look like the three deleted on lines 52-54. Would you agree lines 60-62 should be deleted?

File: cleaners/miro.xml [hunk 4]
```diff
@@ -18,26 +18,48 @@
...
     <action command="delete" search="file" path="~/.miro/miro-log"/>
     <action command="delete" search="file" path="~/.miro/miro-log.1"/>
-    <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro-downloader.log"/>
-    <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro.log"/>
-    <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro.log.1"/>
-    <action command="delete" search="file" path="$USERPROFILE\Local Settings\Temp\Miro_Download_Daemon_a.txt"/>
+    <action command="delete" search="file" path="$Temp\Miro-downloader.log"/>
+    <action command="delete" search="file" path="$Temp\Miro.log"/>
+    <action command="delete" search="file" path="$Temp\Miro.log.1"/>
+    <action command="delete" search="file" path="$Temp\Miro_Download_Daemon_a.txt"/>
```
Comment (az0) [inline cleaners/miro.xml]: Line 55 deletes the same file (Miro_Download_Daemon_a.txt) deleted on line 63. Would you agree line 55 should be deleted?

File: cleaners/miro.xml (hunk 3 above)
Comment (Tobias-B-Besemer) [reply cleaners/miro.xml]: IMHO, no! One it's the option "temp", one is the option "log".

File: cleaners/miro.xml (hunk 4 above)
Comment (Tobias-B-Besemer) [reply cleaners/miro.xml]: I would let it...

File: cleaners/miro.xml (hunk 2 above)
Comment (Tobias-B-Besemer) [reply cleaners/miro.xml]: I do my best! ;-)

File: cleaners/miro.xml (hunk 3 above)
Comment (az0) [reply cleaners/miro.xml]: I think a good principle is the same action shouldn't be repeated in the same cleaner file (either within the same option or in a different option). First, if there are options labeled "log" and "temporary files," it implies chosing one doesn't do the other. If you think for Miro they are similar, you could eliminate the "logs" option and put all the actions under "temporary files."

Comment (Tobias-B-Besemer) [reply cleaners/miro.xml]: I prefer to remove "Temporary files", again. I will do this.

**Pull request merged**